odoo/addons/base/tests/test_views.py

```diff
@@ -4567,7 +4567,7 @@ def test_uncommented_invisible_field(self):
             'helpdesk_repair',
         )
 
-        modules_without_error = set(only_log_modules)
+        modules_without_error = set(self.env['ir.module.module'].search([('state', '=', 'intalled'), ('name', 'in', only_log_modules)]).mapped('name'))
```

Comment:
```suggestion
        modules_without_error = set(self.env['ir.module.module'].search([('state', '=', 'installed'), ('name', 'in', only_log_modules)]).mapped('name'))
```

Hello @Xavier-Do , I just found this completely by chance while looking into why my comment on a field with `invisible="True"` wasn't being taken into account...
It looks like there's a small typo here 😇 
As far as I know, I don't have rights on `odoo-dev`, so I just wanted to let you know :)
Have a nice day!

Reply:
Thanks, I will fix it here https://github.com/odoo/odoo/pull/228977